pallets/gear/src/manager/mod.rs

```diff
@@ -239,7 +239,13 @@ where
         })
     }
 
-    pub fn set_program(&self, program_id: ProgramId, code_info: &CodeInfo, message_id: MessageId) {
+    pub fn set_program(
+        &self,
+        block_num: u32,
```

Comment:
Better to use the same name as in other places.
```suggestion
        block_number: u32,
```

Reply:
agree with @shamilsan

Reply:
done